.github/ISSUE_TEMPLATE/template-incubation-application.md

```diff
@@ -0,0 +1,251 @@
+# $PROJECT Incubation Application
+
+This template provides the project with a framework to inform the TOC of their conformance to the Incubation Level Criteria. 
+
+Project Repo(s): $URL
+Project Site:  $URL
+Sub-Projects: $LIST
+Communication: $SLACK
+
+Project points of contacts: $NAME, $EMAIL
+
+## Incubation Criteria Summary for $PROJECT
+
+### Adoption Assertion
+
+_The project has been adopted by the following organizations in a testing and integration or production capacity:_
+* 
+
+## Application Process Principles
+
+### Suggested
+
+N/A
+
+### Required
+
+- [ ] **Give a presentation and engage with the domain specific TAG(s) to increase awareness**
+  - This was completed and occurred on DD-MMM-YYYY, and can be discovered at $LINK.
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **TAG provides insight/recommendation of the project in the context of the landscape**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **All project metadata and resources are [vendor-neutral](https://contribute.cncf.io/maintainers/community/vendor-neutrality/).**
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Review and acknowledgement of expectations for [Sandbox](sandbox.cncf.io) projects and requirements for moving forward through the CNCF Maturity levels.**		
+- Met during Project's application on DD-MMM-YYYY.
+
+<!-- (Project assertion goes here) --> 
+
+- [ ] **Due Diligence Review.**
+
+Completion of this due diligence document, resolution of concerns raised, and presented for public comment satisifies the Due Diligence Review criteria.
```

Comment:
> this due diligence document

Question: Google docs like [this (Karmada)](https://docs.google.com/document/d/1dm9mMANZ13UGTiNbW8DaDvrmuaAAJhHIoJmisQ6lUuI/edit#heading=h.17q6bz2003df) are no longer required?

Reply:
Correct, this template replaces that. (with Great Rejoicing!)